app/controllers/application_controller.rb

```diff
@@ -1,18 +1,18 @@
 class ApplicationController < ActionController::Base
   helper_method :current_user
-  before_action :require_user
+  # before_action :require_user
 
   private
 
   def current_user
     @current_user ||= UserFacade.user_get(session[:user_id]) if session[:user_id]
   end
 
-  def require_user
-    permission_denied unless current_user
-  end
-
-  def permission_denied
```

Comment:
Is there a reason this isn't just added into #require_user?

Reply:
Yes, right now it creates an infinite loop, so we need to adjust something there